estate/models/building_type_model.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class building_type_model(models.Model):
```

Comment:
The class name should match the file name (for most cases) and follow the PascalCase convention.
As PEP8 says (https://peps.python.org/pep-0008/#class-names): 
> Class names should normally use the CapWords convention

```suggestion
class BuildingType(models.Model):
```